content/documentation/pages/5-feature-guides/2-batch/1-deployment-properties.md

```diff
@@ -4,4 +4,42 @@ title: 'Deployment Properties'
 description: 'How to set deployment properties'
 ---
 
-Introduction to deployment properties
+# Deployment Properties
+
+When task definitions are launched to the target platforms `Local`, `CloudFoundry` and `Kubernetes`, you can provide the configuration properties that are applied to the task applications at launch time.
+For instance you can specify:
+
+- Deployer Properties - These properties customize how tasks are launched.
+- Application Properties - These are application specific properties.
+
+## Deployer Properties
+
+Deployer properties are those properties that tell Spring Cloud Data Flow's deployer how the application should be launched.
+They are in the format of `deployer.<application name>.property`.
+To set the maximum heap to 2048m for this Task launch, would require the following deployer property, `deployer.timestamp.local.javaOpts=-Xmx2048m`.
```

Comment:
This only will work in Local, though. Perhaps qualify it here? It might also help to point to the other available properties in the ref. guide.